src/builtins/amp-pixel/amp-pixel.js

```diff
@@ -18,6 +18,10 @@ export class AmpPixel extends BaseElement {
 
     /** @private {?Promise<!Image>} */
     this.triggerPromise_ = null;
+
+    /** @private @const {boolean} */
+    this.isAttributionReportingSupported_ =
```

Comment:
I just made #38620 where I try to pull this util out to be shared. Once I get that in we can also use here.

Reply:
Thanks! Let me know when that change is made and I'll update this.

Reply:
This was merged today so you should be able to use it. Note you may have to allowlist the dep check (see my PR)

Reply:
If they pass you should be good.